tools/scripts/updateDocumentSnippets.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import os.path
+import subprocess
+
+# ---------------------------------------------------
+# Update code examples
+# ---------------------------------------------------
+# For info on mdsnippets, see https://github.com/SimonCropp/MarkdownSnippets
+
+# install dotnet SDK from http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
+# Then install MarkdownSnippets.Tool with
+#   dotnet tool install -g MarkdownSnippets.Tool
+# To update:
+#   dotnet tool update  -g MarkdownSnippets.Tool
+# To uninstall (e.g. to downgrade to a lower version)
+# dotnet tool uninstall -g MarkdownSnippets.Tool
+
+assert os.path.exists('mdsnippets.json'), 'This script must be run from the root of Catch2 source tree'
```

Comment:
One of the script files already has a way to find out the base path to the repository -- I think this script should use it to ensure its working dir is at the repo's root.

Reply:
Good point - done.